doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -167,112 +213,82 @@ maximize the impact of the mclock scheduler.
-   :confval:`bluestore_throttle_deferred_bytes` to 32 KiB(32768 Bytes) each to begin
-   with. Otherwise, you may skip to the next section.
-10. Run the 4KiB random write workload as before on the OSD(s) for 300 secs.
-11. Note the overall throughput from the cbt log files and compare the value
-    against the baseline throughput in step 8.
-12. If the throughput doesn't match with the baseline, increment the bluestore
-    throttle options by 2x and repeat steps 9 through 11 until the obtained
-    throughput is very close to the baseline value.
-
-For example, during benchmarking on a machine with NVMe SSDs, a value of 256 KiB for
-both bluestore throttle and deferred bytes was determined to maximize the impact
-of mclock. For HDDs, the corresponding value was 40 MiB, where the overall
-throughput was roughly equal to the baseline throughput. Note that in general
-for HDDs, the bluestore throttle values are expected to be higher when compared
-to SSDs.
-
-.. _cbt: https://github.com/ceph/cbt
+  :confval:`bluestore_throttle_bytes` and
+  :confval:`bluestore_throttle_deferred_bytes`. But these parameters may also be
+  determined during the benchmarking phase as described below.
 
+Benchmarking Test Steps Using OSD Bench
+```````````````````````````````````````
 
-Specifying  Max OSD Capacity
-----------------------------
+The steps below use the default shards and detail the steps used to determine
+the correct bluestore throttle values.
 
-The steps in this section may be performed only if the max osd capacity is
-different from the default values (SSDs: 21500 IOPS and HDDs: 315 IOPS). The
-option ``osd_mclock_max_capacity_iops_[hdd, ssd]`` can be set by specifying it
-in either the **[global]** section or in a specific OSD section (**[osd.x]** of
-your Ceph configuration file).
+1. Bring up your Ceph cluster and login to the Ceph node hosting the OSDs that
+   you wish to benchmark.
+2. Ensure that the bluestore throttle options ( i.e.
+   :confval:`bluestore_throttle_bytes` and
+   :confval:`bluestore_throttle_deferred_bytes`) are set to the default values.
+3. Run a simple 4KiB random write workload on an OSD using the following
+   commands:
 
-Alternatively, commands of the following form may be used:
+  .. note:: Note that before running the test, caches must be dropped to get an
+            accurate measurement.
 
   .. prompt:: bash #
 
-     ceph config set [global, osd] osd_mclock_max_capacity_iops_[hdd,ssd] <value>
-
-For example, the following command sets the max capacity for all the OSDs in a
-Ceph node whose underlying device type is SSDs:
+    ceph tell osd.x cache drop
 
   .. prompt:: bash #
 
-    ceph config set osd osd_mclock_max_capacity_iops_ssd 25000
-
-To set the capacity for a specific OSD (for example "osd.0") whose underlying
-device type is HDD, use a command like this:
-
-  .. prompt:: bash #
+    ceph tell osd.x bench 12288000 4096 4194304 200
```

Comment:
might be useful to describe what these parameters mean, not sure if this command is very well documented, if so, we could add a reference here

Reply:
Added a reference to the OSD subsystem where the command is described briefly. I also added a small section to describe the command syntax used in the benchmark.